redbot/cogs/general/general.py

```diff
@@ -234,38 +240,210 @@ def __init__(self):
 
     @commands.command()
     @commands.guild_only()
-    async def serverinfo(self, ctx):
-        """Show server information."""
+    @commands.bot_has_permissions(embed_links=True)
+    async def serverinfo(self, ctx, details: bool = False):
+        """
+        Show server information.
+    
+        `details`: Toggle it to `True` to show more informations about this server.
+        Default to False.
+        """
         guild = ctx.guild
+        passed = (ctx.message.created_at - guild.created_at).days
+        created_at = _("Since {date}. That's over {num} days ago!").format(
```

Comment:
I think one of these would be clearer:
```suggestion
        created_at = _("Active since {date}. That's over {num} days ago!").format(
```
```suggestion
        created_at = _("Running since {date}. That's over {num} days ago!").format(
```
```suggestion
        created_at = _("Created on {date}. That's over {num} days ago!").format(
```